circle-mlir/infra/overlay/prepare-venv

```diff
@@ -34,12 +34,36 @@ fi
 
 # NOTE version
 # - https://github.com/onnx/onnx/blob/master/docs/Versioning.md
+# - https://github.com/Samsung/ONE/issues/15226#issuecomment-2829062592
 
 VER_TORCH=2.4.1+cpu
 VER_ONNX=1.17.0
 VER_ONNXRUNTIME=1.18.0
 VER_NUMPY=1.24.3
 
+PYTHON_VERSION=$($VENV_PYTHON -c "import sys; print(f'{sys.version_info.major}.{sys.version_info.minor}')")
+echo "python3 version is $PYTHON_VERSION"
+
+if [[ "$PYTHON_VERSION" == "3.8" ]]; then
+  # Ubuntu 20.04 is python3.8
+  : # use as is
+elif [[ "$PYTHON_VERSION" == "3.10" ]]; then
+  # Ubuntu 22.04 is python3.10
+  VER_TORCH=2.7.0+cpu
+  VER_ONNX=1.17.0
+  VER_ONNXRUNTIME=1.21.1
+  VER_NUMPY=1.26.4
+elif [[ "$PYTHON_VERSION" == "3.12" ]]; then
+  # Ubuntu 24.04 is python3.12
+  VER_TORCH=2.7.0+cpu
+  VER_ONNX=1.17.0
+  VER_ONNXRUNTIME=1.21.1
+  VER_NUMPY=1.26.4
+else
+  echo "Error prepare-venv: Unsupported python version: $PYTHON_VERSION"
+  exit -1
```

Comment:
maybe
```suggestion
  exit 1
```
Exit codes are interpreted as unsigned 8-bit values (0–255). If you write exit -1, the shell wraps that to 255, which is less clear. It’s more conventional to use exit 1 for a generic error, since 1 clearly indicates failure and doesn’t rely on wrapping behavior.